src/Components/CreateImageWizard/formComponents/Repositories.js

```diff
@@ -454,16 +458,24 @@ const Repositories = (props) => {
                               </Button>
                             </Td>
                             <Td dataLabel={'Architecture'}>
-                              {repo.distribution_arch}
+                              {repo.name ? repo.distribution_arch : '-'}
                             </Td>
                             <Td dataLabel={'Version'}>
-                              {repo.distribution_versions}
+                              {repo.name ? repo.distribution_versions : '-'}
+                            </Td>
+                            <Td dataLabel={'Packages'}>
+                              {repo.name ? repo.package_count : '-'}
                             </Td>
-                            <Td dataLabel={'Packages'}>{repo.package_count}</Td>
                             <Td dataLabel={'Status'}>
                               <RepositoriesStatus
-                                repoStatus={repo.status}
+                                repoStatus={
+                                  repo.name ? repo.status : 'Unavailable'
+                                }
                                 repoUrl={repo.url}
+                                repoIntrospections={
+                                  repo.last_introspection_time
+                                }
+                                repoFailCount={repo.failed_introspections_count}
```

Comment:
Is there the check for repo existing missing for the fail count ?

Reply:
`RepositoriesStatus` component checks whether `repoFailCount` is defined and renders the count conditionaly based on that

```
{repoFailCount && (
  <DescriptionListGroup>
  <DescriptionListTerm>
```